README.md

```diff
@@ -92,6 +92,9 @@ exercises some rudimentary business logic of selected extensions.
 `start-stop.native.memory` property - configure memory limits for the run command for the final Native application
 - append for example `-Dstart-stop.native.memory="-Xms96m -Xmx96m"` to the mvn command
 
+`artifact-generator.offering.registry.name` property - configure the registry name (default `testingregistry`) to which the offering should be added.
+- append for example `-Doffering.registry.name="registry.quarkus.redhat.com"` to the mvn command
```

Comment:
```suggestion
- append for example `-Dartifact-generator.offering.registry.name="registry.quarkus.redhat.com"` to the mvn command
```

Reply:
Sorry missed this when copying the command.